components/ActivityIndicator/src/MDCActivityIndicator.m

```diff
@@ -155,7 +155,9 @@ - (void)commonMDCActivityIndicatorInit {
   _strokeWidth = 2.0f;
 
   // Colors.
-  _cycleColors = [MDCActivityIndicator defaultColors];
+  if (![MDCActivityIndicator appearance].cycleColors) {
```

Comment:
UIAppearance a bit strange.  It does some magic under the hood to watch property invocations.  The benefit is we can use it to set our defaults, and if a client decides to use properties it will quietly overwrite the values stored in the UIAppearance proxy.

1. We don't have to check if values have been set on the appearance proxy.
2. We must set our default values on the appearance proxy, not on the properties in our init methods.

So, I would expect something along the following:

+(void)initialize {
  // Assign all default UIAppearance properties here
  [MDCActivityIndicator appearance].cycleColors = [MDCActivitiyIndicator defaultColors];
}

-OR-
We may be able to get rid of the +defaultColors method completely.  (Less code FTW!)

+(void)initialize {
  // Assign all default UIAppearance properties here
  [MDCActivityIndicator appearance].cycleColors = @[
      [UIColor colorWithRed:0.129f green:0.588f blue:0.953f alpha:1],
      [UIColor colorWithRed:0.957f green:0.263f blue:0.212f alpha:1],
      [UIColor colorWithRed:1.0f green:0.922f blue:0.231f alpha:1],
      [UIColor colorWithRed:0.298f green:0.686f blue:0.314f alpha:1]
];
}

This applies here and below.  (I would be surprised to ever query the UIAppearance proxy directly)